### ISSUE TITLE
Wings Reborn Bug

### ISSUE THREAD

**Issue by ventibrot** [Asek3/Oculus#658]
### What happened?

When Oculus is installed the Wings Reborn mod has a small display bug. Some of the wing variants are missing their feathers. The wings are still usable. Any wings with "feathers" are instead plucked to look like chicken wings. The only wings that are fully visible are the insect wings. 
Install: Using Curseforge 1.20.1-47.3.0
Wings Reborn version 1.20.1-2.1.4
Oculus 1.20.1-1.7
Embeddium 1.20.1-0.3.22

To see the bug; in the creative search bar, search wing, 10 potion bottles with different wing variants should appear. Drink the angel wing potion, then change your pov to third person to view the wings. The wings are still functional, but are missing their feathers. Drink the dragon wings to see the same effect, as well as the flame wings. The only wings that have the correct textures are the Monarch Butterfly wings, Slime wings, Blue butterfly wings, and fairy wings.
This issue was brought up to Embeddium as well and they also had the code that was causing the issue. They have fixed it on their end with the 0.3.22 release. You can see the Embeddium Issue report here: https://github.com/embeddedt/embeddium/issues/318 

### Minecraft Version

1.20.1

### Oculus Version

oculus-mc1.20.1-1.7.0

### Rubidium / Embeddium Version

Embeddium 1.20.1-0.3.22

### Operating System

Windows 10

### What is your GPU?

Nvidia GeForce RTX 3060

### Java Version

Java 8

### Additional context

[2024-06-30-1.log.gz](https://github.com/user-attachments/files/16060675/2024-06-30-1.log.gz)

**Comment by Killquota** [comment]
if this gets fixed on oculus' end wings mod with be so widely used

**Comment by WorldsOfRosefire** [comment]
agree, loved those wings on 1.12.2, 1.16.5 and now Wings Reborn on 1.20.1. would be epic if you guys could fix this. please

**Comment by TheChickenWizard** [comment]
would love if this finally got fixed, It's so odd having two worms flapping around instead of actual wings

**Comment by Droid-B1045** [comment]
would be a great mod if we could use shaders with them

**Comment by runicvoid12-code** [comment]
Any update on this?

**Comment by Asgoldenblood** [comment]
ello, I found a way to fix it, using another mod, this mod here: [https://modrinth.com/mod/acceleratedrendering/versions](url)
 I only tested it with version 1.0.7 of it, it worked perfectly, try it
